src/App.tsx

```diff
@@ -35,6 +40,26 @@ import { MediaDevicesProvider } from "./livekit/MediaDevicesContext";
 
 const SentryRoute = Sentry.withSentryRouting(Route);
 
+interface BackgroundProviderProps {
+  children: JSX.Element;
+}
+
+const BackgroundProvider = ({ children }: BackgroundProviderProps) => {
+  const { pathname } = useLocation();
+
+  useEffect(() => {
+    let backgroundImage = "";
+    if (!["/login", "/register"].includes(pathname)) {
```

Comment:
Non-blocking: I think it would be more elegant and React-y to apply the background by putting a CSS class on the specific page-level components that are supposed to have it, rather than inspecting the URL like this. This would also allow the background CSS to be isolated to a single CSS module file, which I think would be desirable.